File: partnersapi.py
import requests
import json


def getUrl(url):
    link = requests.get(url)
    dictionary = json.loads(link.text)
    return dictionary

dictionary = getUrl('http://join.navgurukul.org/api/partners')
with open('partners.json','w') as f:
    json.dump(dictionary,f)

data = dictionary['data']

def nameNgo():
    for i in data:
        print(i['name'])

nameNgo()

File: partnersapi.js
// const axios = require('axios');

// axios.get('http://join.navgurukul.org/api/partners')
//   .then(response => {
//     console.log(response.data);
//     // console.log(response.data.explanation);
//   })
//   .catch(error => {
//     console.log(error);
//   });

const request = require('request');

request('http://join.navgurukul.org/api/partners', { json: true }, (err, res, body) => {
  if (err){
    return err
  }
  else{
    data = body['data']
    for (var i = 0; i < data.length; i++){
      console.log(data[i]['name'])
    }
  }
});

